src/Microsoft.Azure.WebJobs.Host/Queues/QueueProcessorFactoryContext.cs

```diff
@@ -32,17 +35,19 @@ public QueueProcessorFactoryContext(CloudQueue queue, TraceWriter trace, CloudQu
             Queue = queue;
             PoisonQueue = poisonQueue;
             Trace = trace;
+            Logger = loggerFactory?.CreateLogger(LoggingCategories.CreateTriggerCategory("Queue"));
```

Comment:
I don't think there's any logging from other triggers. I want someone to be able to filter to *only* see Queue triggers if they're trying to debug something. So you'd turn on Trace-level (very verbose) logging for your Queues and then you could look through those logs without getting swamped.